app/src/main/java/org/sopt/at/views/history/components/pager/HistoryTabPagerScreen.kt

```diff
@@ -0,0 +1,110 @@
+package org.sopt.at.views.history.components.pager
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.navigation.NavController
+import org.sopt.at.R
+import org.sopt.at.components.dialogs.DialogType
+import org.sopt.at.views.history.components.tab.HistoryTabData
+import org.sopt.at.models.history.HistoryEmptyEntity
+import org.sopt.at.viewmodels.HistoryViewModel
+import org.sopt.at.views.history.components.itemgrid.HistoryLazyGrid
+import org.sopt.at.views.navigation.Screen
+import org.sopt.designsystem.theme.MyAtSoptTheme
+
+@Composable
+fun HistoryTabPagerScreen(
+    modifier: Modifier = Modifier,
+    navController: NavController,
+    emptyData : HistoryEmptyEntity?,
+    historyViewModel: HistoryViewModel = hiltViewModel()
+) {
+    val tab by historyViewModel.historyTab.collectAsState()
+    val historyList by historyViewModel.historyDataList.collectAsState(initial = emptyList())
+
+
+    if (historyList.isEmpty() || tab != HistoryTabData.TAB_SERIES) {
+        Column(
+            modifier = modifier
+                .fillMaxSize()
+                .background(MyAtSoptTheme.colors.white)
+                .padding(16.dp)
+        ) {
+            Text(
+                text = emptyData?.title.toString(),
```

Comment:
nullable 타입에서 title이 null이라면 "null"이 텍스트에 나오게 됩니다.
빈 문자열이 보여야한다면 
```suggestion
                text = emptyData?.title ?: ""
```
요렇게 해봐도 좋을 것 같아요.